backend/src/common/interfaces/service.interface.ts

```diff
@@ -1,5 +1,5 @@
 type Service<T = unknown> = {
-    find(): Promise<T>;
+    find(id: number): Promise<T>;
```

Comment:
Name this method findById
```suggestion
    findById(id: number): Promise<T>;
```

Reply:
To clarify this one, keep find method and define a new method findById?

Reply:
Yes

Reply:
This is done now. Retained find method and defined separate findById method.